src/Data/V/Linear/Internal.hs

```diff
@@ -112,32 +112,30 @@ class (n ~ Arity b f) => Elim n a b f | n a b -> f, f b -> n where
   -- > elim @2 :: (a %1 -> a %1 -> b) %1 -> V 2 a %1 -> b
   -- > elim @3 :: (a %1 -> a %1 -> a %1 -> b) %1 -> V 3 a %1 -> b
   elim :: f %1 -> V n a %1 -> b
+  elim f v = elim' f v
+  {-# INLINEABLE elim #-}
```

Comment:
For obscure reasons, the `{-# INLINABLE #-}` pragma is necessary for `V.elim` (but not `Replicator.elim`). Also, neither `INLINE` nor `NOINLINE` work in this context. It seems that the contexts where something `INLINABLE` gets inlined are not completely contained in the contexts where something `INLINE` gets inlined.